test/functional/ex_cmds/encoding_spec.lua

```diff
@@ -25,7 +25,7 @@ describe('&encoding', function()
     clear('--cmd', 'set enc=latin1')
```

Comment:
The "after setup" and "before startup" test descriptions need to be updated. 

Also noticed that `e_afterinit` is not used, though it should be for 'termguicolors'.